kythe/proto/storage.proto

```diff
@@ -91,7 +91,7 @@ message VNameMask {
   bool language = 5;
 }
 
-// An Entry associates a fact with a graph object (node or edge).  This is the
+// An Entry associates a fact with a graph object (node XOR edge).  This is the
```

Comment:
We should probably clean this up further.  Originally, all Entry messages were considered "facts", but now we tend to refer to an Entry as either a fact XOR an edge and a group of them sharing the same source VName as a node.

Reply:
new wording